packages/@angular/cli/models/webpack-configs/production.ts

```diff
@@ -74,7 +74,8 @@ export const getProdConfig = function (wco: WebpackConfigOptions) {
       new webpack.optimize.UglifyJsPlugin(<any>{
         mangle: { screw_ie8: true },
         compress: { screw_ie8: true, warnings: buildOptions.verbose },
-        sourceMap: buildOptions.sourcemap
+        sourceMap: buildOptions.sourcemap,
+        comments: false
```

Comment:
I verified with legal and they said removing the comments from a build output is fine in our case. This LGTM (needs to fix errors on CI)